snmp/datadog_checks/snmp/data/default_profiles/dell.yaml

```diff
@@ -1,15 +0,0 @@
-# Profile for Dell  devices
-
-extends:
-  - _base.yaml
-  - _generic-ip.yaml
-  - _generic-tcp.yaml
-  - _generic-udp.yaml
-
-metadata:
-  device:
-    fields:
-      vendor:
-        value: "dell"
-
-sysobjectid: 1.3.6.1.4.1.674.*
```

Comment:
those `_generic-*` are only used in one profile -- `dell-poweredge`, so I created base profile for dell profiles with _base.yaml, which is common for every profile.